cpp/include/cuspatial/experimental/detail/algorithm/is_point_in_polygon.cuh

```diff
@@ -100,5 +93,30 @@ __device__ inline bool is_point_in_polygon(Cart2d const& test_point,
 
   return point_is_within;
 }
+
+/**
+ * @brief Compatibility layer with non-OOP style input
```

Comment:
Is this compatibility layer needed somewhere?

Reply:
In all existing brute force point_in_polygon APIs.